doc/nrf/ug_matter_configuring_protocol.rst

```diff
@@ -0,0 +1,75 @@
+.. _ug_matter_configuring_protocol:
+
+Configuring Matter in |NCS|
+###########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This page describes what is needed to start working with Matter in the |NCS|.
+
+.. _ug_matter_configuring_requirements:
+
+Required components for Matter network
+**************************************
+
+The Matter network requires the following components to properly operate:
+
+* Matter controller - configured either on PC or mobile
+* Thread Border Router - required by the Matter controller
```

Comment:
```suggestion
* Thread Border Router - configured either on PC or Raspberry Pi
```
It's required not only by the controller. It's required by all Thread nodes, so maybe it's not worth mentioning that.

Reply:
@greg-fer You liked the suggestion, but didn't commit it. Intentionally? :)

Reply:
I edited the wrong line :/

Reply:
@Damian-Nordic , please recheck. Pushed a commit fixing this.